package.json

```diff
@@ -11,6 +11,9 @@
   "scripts": {
     "start": "gulp & node ./bin/www",
     "dev": "gulp dev",
+    "lint": "eslint --quiet **/**/*.{js,css,scss,pug}",
+    "lint:fix": "eslint --fix .",
```

Comment:
This command is not working as intended. It still logs this -
![Screenshot 2020-03-24 at 4 37 56 PM](https://user-images.githubusercontent.com/31413407/77419117-e67f4a80-6ded-11ea-9716-d6322b35a53c.png)

Reply:
I will update the changes.